jobs/raghava_nelbo_trello_to_gold.py

```diff
@@ -0,0 +1,71 @@
+"""
+This script will fetch all the cards of a trello board that according to the below criteria:
+- present in doing list
+- does not have any activity from past specified days
+Save the data to Delta Lake Gold table
+This script is scheduled to run twice a week
+"""
+
+import os
+import sys
+from datetime import datetime, timedelta
+import argparse
+from loguru import logger
+from pyspark.sql.functions import col
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers import spark_helper as sh, common_functions as cf, trello_functions as tf
+from config import delta_table_conf as dc
+
+def get_activity_cards(start_date,end_date):
+    """
+    Fetch all the cards data between provided date range and
+    active from past specified days and save it into a Delta Lake Gold table
+    :param spark: Spark session
+    :param board_id: trello board id
+    :param board_name: trello board name
+    :param no_of_days: number of days the cards are not active
+    :return none

```

Comment:
The description needs to be changed. It is that of noactivity

Reply:
removed unwanted lines